### PR TITLE
build: update optimizer

### PR DESCRIPTION
**Pull request by MerlinEgalite**

### CODE REVIEW

File: foundry.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ fs_permissions = [
     { access = "read", path = "./lib/morpho-blue/out/"}
 ]
 libs = ["lib"]
-optimization_runs = 999999 # Etherscan does not support verifying contracts with more optimization runs.
+optimizer_runs = 999999 # Etherscan does not support verifying contracts with more optimization runs.
```
Comment (QGarchery) [inline foundry.toml:11]:
```suggestion
optimizer_runs = 999999 # Etherscan does not support verifying contracts with more optimizer runs.
```
let's say that now, so less likely we do a mistake in the future

**Review by QGarchery — APPROVED** [review]

File: foundry.toml [hunk 2]
```diff
@@ -8,7 +8,7 @@ fs_permissions = [
     { access = "read", path = "./lib/morpho-blue/out/"}
 ]
 libs = ["lib"]
-optimization_runs = 999999 # Etherscan does not support verifying contracts with more optimization runs.
+optimizer_runs = 999999 # Etherscan does not support verifying contracts with more optimizer runs.
```
Comment (QGarchery) [inline foundry.toml:11]: Number to change, so probably the description too

File: foundry.toml
Comment (QGarchery) [inline foundry.toml:1]: Same remark as [this comment](https://github.com/morpho-org/metamorpho/pull/378#discussion_r1440224667) about the outdated link at the end

**Review by QGarchery — CHANGES_REQUESTED** [review]

**Review by Jean-Grimal — APPROVED** [review]

**Review by QGarchery — APPROVED** [review]

**Review by MathisGD — COMMENTED** [review]
LGTM

**Comment by Rubilmax** [comment]
What was the rationale behind decreasing the optimization runs?

**Comment by MerlinEgalite** [comment]
*increase

`optimization_runs` does not work, we must use `optimizer_runs` instead

**Comment by Rubilmax** [comment]
We did decrease from 999,999 to 80,000 ^^ otherwise understood!

**Comment by MerlinEgalite** [comment]
> We did decrease from 999,999 to 80,000 ^^ otherwise understood!

Yes but it was not working so in practice we were compiling with 200 😬

**Pull request merged**